app/src/ui/lfs/attribute-mismatch.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react'
+import { Button } from '../lib/button'
+import { ButtonGroup } from '../lib/button-group'
+import { Dialog, DialogContent, DialogFooter } from '../dialog'
+
+interface IAttributeMismatchProps {
+  /** Called when the dialog should be dismissed. */
+  readonly onDismissed: () => void
+
+  /** Called when the user has chosen to replace the update filters. */
+  readonly onUpdateExistingFilters: () => void
+}
+
+export class AttributeMismatch extends React.Component<
+  IAttributeMismatchProps,
+  {}
+> {
+  public render() {
+    return (
+      <Dialog
+        id="lfs-attribute-mismatch"
+        title={
+          __DARWIN__ ? (
```

Comment:
Why the parenthesis here?

Reply:
That's all Prettier's doing 🙄

Reply:
I'm starting to think prettier ~doesn't match my opinions~ is too opinionated.

Reply:
@iAmWillShepherd Open an issue on prettier/prettier then 😄

Reply:
I can see that going over well! lol Issue title: **I disagree with your opinions, my opinions are more correct than your opinions, so change your opinions to be in sync with my opinions.** I'll adapt.